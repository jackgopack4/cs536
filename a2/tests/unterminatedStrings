"hello
  "
    "this is a sample string \n with a newline in the middle"
      "that previous string
        " "should have\n said unterminated string literal ignored"
          " test \ "
"unterminated
"also unterminated \"
"backslash followed by space: \ is not allowed"
"bad escaped character: \a AND not terminated
"\n\t\'\"\\"
"\n\n\n\n\a\n\n\"
"\n\t\'\"\\
