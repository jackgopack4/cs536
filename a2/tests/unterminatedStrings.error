1:1 ***ERROR*** unterminated string literal ignored
2:3 ***ERROR*** unterminated string literal ignored
4:7 ***ERROR*** unterminated string literal ignored
5:23 ***ERROR*** illegal character ignored: \
5:66 ***ERROR*** unterminated string literal ignored
6:11 ***ERROR*** string literal with bad escaped character ignored
7:1 ***ERROR*** unterminated string literal ignored
8:1 ***ERROR*** unterminated string literal ignored
9:1 ***ERROR*** string literal with bad escaped character ignored
10:1 ***ERROR*** unterminated string literal with bad escaped character ignored
12:1 ***ERROR*** unterminated string literal with bad escaped character ignored
13:1 ***ERROR*** unterminated string literal ignored
