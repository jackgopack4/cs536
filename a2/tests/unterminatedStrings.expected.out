Token: STRINGLITERAL
LineNum: 3
CharNum: 5
Value: "this is a sample string \n with a newline in the middle"

Token: STRINGLITERAL
LineNum: 5
CharNum: 9
Value: " "

Token: ID
LineNum: 5
CharNum: 12
Value: should

Token: ID
LineNum: 5
CharNum: 19
Value: have

Token: ID
LineNum: 5
CharNum: 24
Value: n

Token: ID
LineNum: 5
CharNum: 26
Value: said

Token: ID
LineNum: 5
CharNum: 31
Value: unterminated

Token: ID
LineNum: 5
CharNum: 44
Value: string

Token: ID
LineNum: 5
CharNum: 51
Value: literal

Token: ID
LineNum: 5
CharNum: 59
Value: ignored

Token: STRINGLITERAL
LineNum: 11
CharNum: 1
Value: "\n\t\'\"\\"

